docs/components/PresentCredentialOptions.yml

```diff
@@ -34,6 +37,7 @@ components:
           description: The intended domain of validity for the proof. For example website.example
       example:
         {
+          "type": "https://w3id.org/security#Ed25519Signature2018",
```

Comment:
Same issue as above.

Reply:
Amended to use the compact form, for consistency.
https://github.com/w3c-ccg/vc-http-api/compare/322fba6..173c26b0df3ca1a1406f7b8d4469ef5f15b3afcb